### ISSUE TITLE
React search component

### ISSUE THREAD

**Issue by DawChihLiou** [tantaraio/voy#8]

**Comment by markhughes** [comment]
It would be cool if there was a `voy/react` entry point that exported a `VoyProvider` component and a `useVoy` hook

**Comment by DawChihLiou** [comment]
Great idea @markhughes! What do you think about a similarity search bar component?

**Comment by markhughes** [comment]
Absolutely! Adding good quality starting-point components is the perfect way to have anyone get onboard. 

Could follow the headless style that is done in tanstack?

**Comment by DawChihLiou** [comment]
Awesome! Could you point me to the headless style? I'm not familiar with the term in the context of component. Thanks!
